app/screens/home-screen/home-screen.tsx

```diff
@@ -143,7 +143,8 @@ export const HomeScreen: React.FC = () => {
 
   const activateWallet = () => {
     setModalVisible(false)
-    navigation.navigate("phoneFlow")
+    // fixes a screen flash from closing the modal to opening the next screen
```

Comment:
did this commit not fix this issue? https://github.com/GaloyMoney/galoy-mobile/commit/1b8d42e52981f6f7317b6f89b6ef00500a22af43